script.js

```diff
@@ -267,16 +268,31 @@ function createCountiesCartoURI() {
 
 function createStatesCartoURI() {
   const query = `SELECT
-  s.the_geom, s.state_name as municipality, m.policy_type, m.policy_summary, m.link,
+  s.the_geom, s.name as municipality, m.policy_type, m.policy_summary, m.link,
   CASE m.passed WHEN true THEN 'Yes' ELSE 'No' END as passed
-  FROM state_5m s
+  FROM public.states_and_provinces_global s
   INNER JOIN ${cartoSheetSyncTable} m
-  ON s.state_name = m.state
+  ON s.name = m.state
   AND m.admin_scale = 'State'`;
```

Comment:
@river-honer  If you add this to the query it will work.  I reviewed the ISO codes to make sure they join correctly.

```
function createStatesCartoURI() {
  const query = `SELECT
  s.the_geom, s.state_name as municipality, m.policy_type, m.policy_summary, m.link,
  CASE m.passed WHEN true THEN 'Yes' ELSE 'No' END as passed
  FROM state_5m s
  INNER JOIN ${cartoSheetSyncTable} m
  ON s.state_name = m.state
  AND m.admin_scale = 'State'
  AND m.ISO = s.sr_adm0_a3`;

  return `https://ampitup.carto.com/api/v2/sql?q=${query}&format=geojson`;
} 
```

Reply:
thanks @timhitchins !

Reply:
Just making this comment for additional context and in case anyone is wondering why Québec wasn't rendering.  This is because it was spelled Quebec in the moratoriums table instead of Québec.  I updated it in the edit table and the join should work once the new data is added to dropbox (and synced to CARTO).  No additional code changes required from you, I am just giving you heads up.

Reply:
Great. Query updated as well. Thanks @timhitchins for looking in to this deeper.